src/quo/components/profile/collectible_list_item/view.cljs

```diff
@@ -34,16 +60,20 @@
   [rn/view {:style (style/loading-message theme)}])
 
 (defn- loading-image
-  [{:keys [theme gradient-color-index]}]
-  [gradients/view
-   {:theme           theme
-    :container-style (style/loading-image theme)
-    :color-index     gradient-color-index}])
+  [{:keys [theme gradient-color-index loader-opacity]}]
+  [reanimated/view
+   {:style (reanimated/apply-animations-to-style
+            {:opacity loader-opacity}
+            (style/loading-image theme))}
+   [gradients/view
+    {:theme           theme
+     :container-style (style/loading-image theme)
+     :color-index     gradient-color-index}]])
 
 (defn- card-details
-  [{:keys [community? avatar-image-src collectible-name theme state set-state]}]
+  [{:keys [community? avatar-image-src collectible-name theme state set-state image-opacity]}]
   [rn/view {:style style/card-details-container}
-   (cond (not (:avatar-loaded? state))
+   (cond (and not community? (not (:avatar-loaded? state)))
```

Comment:
Shouldn't this `not` be inside paranthesis `()` ?

Reply:
yes, will double check this. Thanks!

Reply:
addressed this, thanks for spotting!